lib/gui/app/components/drive-selector/DriveSelectorModal.jsx

```diff
@@ -0,0 +1,320 @@
+            if (shouldContinue) {
+              openExternal(drive.link)
+            } else {
+              setConfirmModal({ open: false })
+            }
+          } catch (error) {
+            analytics.logException(error)
+          }
+        }
+      })
+    }
+  }
+
+  /**
+   * @summary Select a drive and close the modal
+   * @function
+   * @public
+   *
+   * @param {Object} drive - drive
+   * @param {Function} closeFn - close function
+   * @returns {void}
+   *
+   * @example
+   * selectDriveAndClose({
+   *   device: '/dev/disk2',
+   *   size: 999999999,
+   *   name: 'Cruzer USB drive'
+   * });
+   */
+  const selectDriveAndClose = async (drive, closeFn) => {
+    const canChangeDriveSelectionState = await shouldChangeDriveSelectionState(drive)
+
+    if (canChangeDriveSelectionState) {
+      selectionState.selectDrive(drive.device)
+
+      analytics.logEvent('Drive selected (double click)', {
+        applicationSessionUuid: store.getState().toJS().applicationSessionUuid,
+        flashingWorkflowUuid: store.getState().toJS().flashingWorkflowUuid
+      })
+
+      closeFn()
+    }
+  }
+
+  /**
+   * @summary Memoized getDrives function
+   * @function
+   * @public
+   *
+   * @returns {Array<Object>} - memoized list of drives
+   *
+   * @example
+   * const drives = getDrives()
+   * // Do something with drives
+   */
+  const getDrives = utils.memoize(availableDrives.getDrives, _.isEqual)
+
+  /**
+   * @summary Get a drive's compatibility status object(s)
+   * @function
+   * @public
+   *
+   * @description
+   * Given a drive, return its compatibility status with the selected image,
+   * containing the status type (ERROR, WARNING), and accompanying
+   * status message.
+   *
+   * @returns {Object[]} list of objects containing statuses
+   *
+   * @example
+   * const statuses = getDriveStatuses(drive);
+   *
+   * for ({ type, message } of statuses) {
+   *   // do something
+   * }
+   */
+  const getDriveStatuses = utils.memoize((drive) => {
+    return getDriveImageCompatibilityStatuses(drive, selectionState.getImage())
+  }, _.isEqual)
+
+  /**
+   * @summary Keyboard event drive toggling
+   * @function
+   * @public
+   *
+   * @description
+   * Keyboard-event specific entry to the toggleDrive function.
+   *
+   * @param {Object} drive - drive
+   * @param {Object} evt - event
+   *
+   * @example
+   * <div tabindex="1" onKeyPress="keyboardToggleDrive(drive, evt)">
+   *   Tab-select me and press enter or space!
+   * </div>
+   */
+  const keyboardToggleDrive = (drive, evt) => {
+    const ENTER = 13
+    const SPACE = 32
+    if (_.includes([ ENTER, SPACE ], evt.keyCode)) {
+      toggleDrive(drive)
+    }
+  }
+
+  const hasStatus = hasListDriveImageCompatibilityStatus(selectionState.getSelectedDrives(), selectionState.getImage())
+
+  return (
+    <Modal
+      className='modal-drive-selector-modal'
+      title='Select a Drive'
+      done={close}
+      action='Continue'
+      style={{
```

Comment:
Let's use the default padding if possible

Reply:
It adds a padding to the left which makes the list uncentered